README.md

```diff
@@ -49,54 +77,37 @@ flutter run --dart-define=screenshot=true
 ```
 
 
-## AtomicDEX API library (libmm2.a) version:
+## AtomicDEX API Library Setup:
 
-2.1.10108
-adde21b26
-http://195.201.0.6/dev/
+AtomicDEX Mobile runs the AtomicDEX API locally on the user's device. The API binary is platform-specific and must be manually set up by the developer instead of a typical Flutter dependency.
 
-## Flutter version
+Ensure you run the most recent AtomicDEX API [stable release](https://github.com/KomodoPlatform/atomicDEX-API/releases). Download the API binary for each platform and extract its `libmm2.a` file into the applicable platform's API folder.
 
-Currently using flutter 2.8.1
 
-### Upgrading from 1.22.4
 
-In your flutter directory:
+### [Stable API releases](https://github.com/KomodoPlatform/atomicDEX-API/releases)
+ 
+| API Build | API Path* |
+|--|--|
+| android-aarch64 | `android/app/src/main/cpp/libs/arm64-v8a/libmm2.a` |
+| android-armv7 | `android/app/src/main/cpp/libs/armeabi-v7a/libmm2.a` |
+| iOS | `ios/libmm2.a` |
 
-```
-git checkout 2.8.1
-flutter doctor
-```
-
-In the project directory:
+**Relative to the Flutter project's root folder. E.g. if your name was Bob and you cloned the flutter project into your macOS home directory, the full path for the iOS API would be `/Users/Bob/atomicdex_mobile/ios/libmm2.a`*
 
-```
-flutter clean
-flutter pub get
-```
-
-### beta Flutter
-
-`flutter version` is inconsistent regarding the access to beta versions.
-Git tags can be used instead (that is, when we want to experiment with beta versions of Flutter):
-
-    FD=`which flutter`; FD=`dirname $FD`; FD=`dirname $FD`; echo $FD; cd $FD
-    git pull
-    git reset --hard
-    git checkout -f v1.14.3
+See [our wiki](https://github.com/KomodoPlatform/atomicdex-mobile/wiki/Project-Setup#android-builds-from-scratch) here for more thorough project setup steps. Besides installing the API binary, AtomicDEX Mobile is set up similarly to any other cloned Flutter project.
 
 ### Kotlin vs Flutter
 
-In Android Studio (3.6.2) the latest Kotlin plugin (1.3.71) doesn't work with Flutter “1.12.13+hotfix.7”. To fix it - [uninstall the latest Kotlin](https://github.com/flutter/flutter/issues/52077#issuecomment-600459786) - then the Kotlin version 1.3.61, bundled with the Android Studio, will reappear.
-
+In Android Studio (3.6.2), the latest Kotlin plugin (1.3.71) doesn't work with Flutter “1.12.13+hotfix.7”. To fix it - [uninstall the latest Kotlin](https://github.com/flutter/flutter/issues/52077#issuecomment-600459786) - then the Kotlin version 1.3.61, bundled with the Android Studio will reappear.
```

Comment:
In Android Studio (3.6.2), the latest Kotlin plugin (1.3.71) doesn't work with Flutter `1.12.13+hotfix.7`.
To fix it
 - [uninstall the latest Kotlin](https://github.com/flutter/flutter/issues/52077#issuecomment-600459786) 
 - The Kotlin version 1.3.61, bundled with the Android Studio will reappear.